### ISSUE TITLE
ACDC EventBased is also broken

### ISSUE THREAD

**Issue by amaltaro** [dmwm/WMCore#5767]
For my record :), Julian mentioned ACDCs using EventBased job splitting are creating duplicate lumis. It seems it also creates less jobs than what it should. I'll investigate it

**Comment by amaltaro** [comment]
It seems we need to change how we upload information to the ACDC server, if we want to make it to work for MC from scratch. This is the - important - information currently uploaded:
 fileset['events'] which is the same value as 'RequestNumEvents'. Instead, it should contain EventsPerJob for this job.
 fileset['first_event'] which is always 1.
 fileset['last_event'] which is always 0.
 fileset['runs'] which also contains a list of lumis. Since this can change, it seems useless to me.

If we properly upload the first and last event, then it should be enough to recover the correct events.

**Comment by amaltaro** [comment]
Ok, I did some more progress on this (btw, feel free to assign it to me and don't bother reading it for now :)).

Here is the place that we get the right information about first/last things:
https://github.com/dmwm/WMCore/blob/master/src/python/WMComponent/ErrorHandler/ErrorHandlerPoller.py#L193
e.g.:
```
{'LastRun': 1L, 'FirstRun': 1L, 'inclusivemask': True, 'runAndLumis': {1L: [[529L, 533L]]}, 'LastEvent': 133001L, 'FirstEvent': 132001L, 'LastLumi': 533L, 'FirstLumi': 529L}
```

Then it gets inside this if
https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/ACDC/CouchFileset.py#L106

which uses a "wrong" info from oracle:
https://github.com/dmwm/WMCore/blob/master/src/python/WMComponent/ErrorHandler/ErrorHandlerPoller.py#L354
e.g.:
```
>>> tmpJob['input_files']
[{'lfn': 'MCFakeFile-amaltaro_MonteCarlo_Ext_MonteCarlo_300jobs_testACDC_Alan_150421_155605_676-Production-e9e0b314815afb0e6cd451c154b8d795', 'locations': set(['storage01.lcg.cscs.ch', 'red-srm1.unl.edu']), 'first_event': 1L, 'se_name': 'cms-se.hep.uprm.edu', 'checksums': {}, 'id': 641L, 'size': 0L, 'runs': 'WMCore.DataStructs.Run.Run object', 'last_event': 0, 'newlocations': set([]), 'jobid': 497L, 'newRuns': 'WMCore.DataStructs.Run.Run object', 'parents': set([]), 'events': 250000L, 'merged': 0L}]
```

**Comment by amaltaro** [comment]
Closed by #5860
